### PR TITLE
Add Facebook Android node

### PR DESCRIPTION
**Pull request by AndrewSB**

### CODE REVIEW

**Comment by shahmishal** [comment]
@AndrewSB Email sent for additional information.

**Comment by juri** [comment]
It says OS version is "Ubunutu 16.04 LTS" — sounds like one `u` too much?

**Comment by AndrewSB** [comment]
Thanks for catching that @juri 🙌

File: nodes/x86_64_ubuntu_16_04_LTS.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    "email": "swiftci@fb.com",
+    "company": "Facebook, Inc."
+  },
+  "node": {
+      "platform": "Android",
+      "os_version": "Ubuntu 16.04 LTS"
+  },
+  "jobs": [
+    {
+      "display_name": "Swift - Android (Tools RA, Stdlib RD) (master))",
```
Comment (harlanhaskins) [inline nodes/x86_64_ubuntu_16_04_LTS.json]: I think you've got one too many `)`'s here, too.

Comment (AndrewSB) [reply nodes/x86_64_ubuntu_16_04_LTS.json]: Thanks @harlanhaskins, I must have been really excited when I saw this was released

Comment (harlanhaskins) [reply nodes/x86_64_ubuntu_16_04_LTS.json]: I don’t blame you!

**Comment by DevAndArtist** [comment]
* Any idea why this PR takes that long? 
* And a odd question, what are we trying to achieve to support Swift on Android? 
* Wouldn't it be Google instead of Facebook who would want native support for Swift on Android?

**Comment by shahmishal** [comment]
@AndrewSB Can you please resolve the conflict? Thanks!

**Comment by shahmishal** [comment]
Node: https://ci-external.swift.org/computer/ubuntu-16.04-android/
Job: https://ci-external.swift.org/job/oss-swift-RA-linux-ubuntu-16.04-android/

**Comment by shahmishal** [comment]
@AndrewSB Do you know if this is known failure? 

```
/home/ubuntu/jenkins/workspace/oss-swift-RA-linux-ubuntu-16.04-android/swift/stdlib/public/stubs/LibcShims.cpp:365:9: error: use of undeclared identifier 'getrandom'
      !(getrandom(nullptr, 0, 0) == -1 && errno == ENOSYS);
        ^
/home/ubuntu/jenkins/workspace/oss-swift-RA-linux-ubuntu-16.04-android/swift/stdlib/public/stubs/LibcShims.cpp:367:35: error: use of undeclared identifier 'getrandom'
      actual_nbytes = WHILE_EINTR(getrandom(buf, nbytes, 0));
                                  ^
/home/ubuntu/jenkins/workspace/oss-swift-RA-linux-ubuntu-16.04-android/swift/stdlib/public/stubs/LibcShims.cpp:367:35: error: use of undeclared identifier 'getrandom'
/home/ubuntu/jenkins/workspace/oss-swift-RA-linux-ubuntu-16.04-android/swift/stdlib/public/stubs/LibcShims.cpp:367:23: error: assigning to 'swift::__swift_ssize_t' (aka 'int') from incompatible type 'void'
      actual_nbytes = WHILE_EINTR(getrandom(buf, nbytes, 0));
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/home/ubuntu/jenkins/workspace/oss-swift-RA-linux-ubuntu-16.04-android/swift/stdlib/public/stubs/LibcShims.cpp:353:33: note: expanded from macro 'WHILE_EINTR'
#define WHILE_EINTR(expression) ({                                             \
                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
4 errors generated.
```

**Comment by AndrewSB** [comment]
It isn't, I'm looking now

**Pull request merged**